src/components/createAppointment/timePicker/timePicker.js

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react'
+import { PickerWrapper } from '../createAppointment.styles'
+import { Text } from '../createAppointment.styles'
+import { Listing, Button } from '../timePicker/timePicker.styles'
+
+const data = [
+  '12:00 am',
+  '1:00 pm',
+  '2:00 pm',
+  '3:00 pm',
+  '4:00 pm',
+  '5:00 pm',
+  '6:00 pm',
+  '7:00 pm',
+  '8:00 pm',
+  '9:00 pm',
+]
+
+const TimePicker = ({ onChangeTimeData }) => {
+  const [time, setTime] = useState('')
+  const handleClick = (e) => {
```

Comment:
Why dont you put this event on button? And prevent searching nodeName?

Reply:
I thought it would be better if I have 1 listener on the container who works as event delegation or if I have a lot of listeners in each button.